src/contentTypeConfig.js

```diff
@@ -238,12 +238,34 @@ function getFieldsForContentType(contentType, asQueryString=true) {
     const fields = contentType ? DEFAULT_FIELDS.concat(contentTypeConfig?.[contentType]?.fields ?? []) : DEFAULT_FIELDS;
     return asQueryString ? fields.toString() + ',' : fields;
 }
+/**
+ * Takes the included fields array and returns a string that can be used in a groq query.
+ * @param {Array<string>} filters - An array of strings that represent applied filters. This should be in the format of a key,value array. eg. ['difficulty,Intermediate', 'genre,rock']
+ * @returns {string} - A string that can be used in a groq query
+ */
+function filtersToGroq(filters) {
+    const groq = filters.map(field => {
+            let [key, value] = field.split(',');
+            switch (key) {
+              case 'difficulty':
+                return `&& difficulty_string == "${value}"`;
+              case 'genre':
+                return `&& genre[]->name match "${value}"`;
+              case 'topic':
+                return `&& topic[]->name match "${value}"`;
+              default:
+                return `&& ${key} == "${value}"`;
```

Comment:
you could add here case 'instrumentless' because that too is a special case `return && ${key} == ${value}`;` because value is boolean and not string